examples/03_collateral_management.py

```diff
@@ -0,0 +1,213 @@
+"""
+Collateral Management: Deposits, Withdrawals, and Margin
+
+This example demonstrates:
+1. Viewing current collateral balances
+2. Depositing assets from LightAccount to subaccount
+3. Withdrawing assets from subaccount to LightAccount
+4. Checking margin requirements
+5. Simulating collateral changes (what-if scenarios)
+
+Prerequisites:
+- Funded LightAccount wallet (see 03_bridging_and_funding.py)
```

Comment:
I would add a note that most users really should be using the UI for :
- deposits / withdrawals
- session key registration 

The only reason to do bridging manually is if there's complex cross chain / exchange arb'ing that needs automatic fund movement.